pages/welcome.js

```diff
@@ -0,0 +1,145 @@
+            {LoggedInUser?.collective?.name}
+          </Flex>
+          <Link href="/create">
+            <Container mt="16px" background="#F5FAFF" width={['300px', '472px']} borderRadius="8px">
+              <Flex alignItems="center" p="13px">
+                <Box width="386px">
+                  <Flex fontSize="18px" fontWeight="700" color="black.900" lineHeight="26px">
+                    <FormattedMessage id="collective.create" defaultMessage="Create Collective" />
+                  </Flex>
+                  <Flex fontSize="15px" fontWeight="500" color="black.700" lineHeight="22px" pt="14px">
+                    {formatMessage(messages.createCollectiveDescription)}
+                  </Flex>
+                </Box>
+                <Box pl="39px">
+                  <Image src="/static/images/left-arrow.png" alt="Left Arrow" width="22px" height="20px" />
+                </Box>
+              </Flex>
+            </Container>
+          </Link>
+          <Link href="/organizations/new">
+            <Container mt="16px" width={['300px', '472px']} borderRadius="8px">
+              <Flex alignItems="center" p="13px">
+                <Box width="386px">
+                  <Flex fontSize="18px" fontWeight="700" color="black.900" lineHeight="26px">
+                    <FormattedMessage id="organization.create" defaultMessage="Create Organization" />
+                  </Flex>
+                  <Flex fontSize="15px" fontWeight="500" color="black.700" lineHeight="22px" pt="14px">
+                    {formatMessage(messages.createOrgDescription)}
+                  </Flex>
+                </Box>
+                <Box pl="39px">
+                  <Image src="/static/images/left-arrow.png" alt="Left Arrow" width="22px" height="20px" />
+                </Box>
+              </Flex>
+            </Container>
+          </Link>
+          <Link href="/search">
+            <Container mt="16px" width={['300px', '472px']} borderRadius="8px">
+              <Flex alignItems="center" p="13px">
+                <Box width="386px">
+                  <Flex fontSize="18px" fontWeight="700" color="black.900" lineHeight="26px">
+                    {formatMessage(messages.engageHeading)}
+                  </Flex>
+                  <Flex fontSize="15px" fontWeight="500" color="black.700" lineHeight="22px" pt="14px">
+                    {formatMessage(messages.engageDescription)}
+                  </Flex>
+                </Box>
+                <Box pl="39px">
+                  <Image src="/static/images/left-arrow.png" alt="Left Arrow" width="22px" height="20px" />
+                </Box>
+              </Flex>
+            </Container>
+          </Link>
+          <Flex justifyContent="space-between" width={['100%', '472px']} pl="13px" pr="13px" pb="32px" pt="40px">
+            <StyledLink href={`${LoggedInUser?.collective?.slug}/admin`}>
```

Comment:
I get a broken link while loading. We can't display this if loading the logged in user

![image](https://user-images.githubusercontent.com/1556356/170993787-c497c2d4-781a-43ea-904f-8a0ceaf2b73e.png)

Reply:
Thanks. Added a condition sot that the `Go to settings` links only appears after the logged in use is loaded.